.ci/google-java-format.sh

```diff
@@ -19,101 +37,54 @@ else
   exit 1
 fi
 
+echo "Checking that all excluded java files have same size as matching InputFormatted* file:"
+MISMATCH_CONTENT=$(cat "${NONFORMATTED_LIST[@]}" \
+  | while read -r path; do
+    size=$(sed -E 's|^[[:space:]]+||; s|//.*||' "$path" | wc -c)
+    name=$(echo "$path" | sed -E 's/.*Input([^\.]+)\.java.*/\1/')
+    formatted_path=$(find ./src -type f -name "InputFormatted${name}.java")
+    size_formatted=$(sed -E 's|^[[:space:]]+||; s|//.*||' "$formatted_path" | wc -c)
+    diff=$(( size - size_formatted ))
+    if (( diff < 0 )); then
+      diff=$(( -diff ))
+    fi
+    if (( diff > 500 )); then
+      if ! is_suppressed "$SUPPRESS_GT_500" "Input${name}.java"; then
+        echo "Difference in Input${name}.java and InputFormatted${name}.java is more than 500 bytes ${diff}"
+      fi
+    elif (( diff > 100 )); then
+      if ! is_suppressed "$SUPPRESS_GT_100_LTE_500" "Input${name}.java"; then
+        echo "Difference in Input${name}.java and InputFormatted${name}.java is more than 100 bytes ${diff}"
+      fi
+    elif (( diff > 50 )); then
+      if ! is_suppressed "$SUPPRESS_LTE_100" "Input${name}.java"; then
+        echo "Difference in Input${name}.java and InputFormatted${name}.java is more than 50 bytes ${diff}"
+      fi
+    fi
+  done)
+
+if [[ $(echo -n "$MISMATCH_CONTENT" | wc --chars) -eq 0 ]]; then
```

Comment:
This condition should check some flag that is triggered by decisions at `if ! is_suppressed...`

Reply:
By the way , why we dropped limit 50?
50 bytes was good limit for majority.

Reply:
Printing something in echo is not bound to `exit 1`